app/components/navs/nav-rail-common-item.jsx

```
@@ -89,8 +88,8 @@ export default function NavRailCommonItem(
         props.href = '#';
     }
     return (
-        <div className={`nav-item w-full rounded-full cursor-pointer font-medium selected-${props.selected}`} role='link'>
-            <Link className={`state-layer w-full relative flex extended-gap-${props.extended} items-center justify-between rounded-full padding-${props.extended ? "extended" : "collapsed"}`} href={props.href}>
+        <div className={`nav-item w-full rounded-full cursor-pointer font-medium selected-${props.selected} transition-all duration-300 ease-in-out`} role='link'>
```

Biome (1.9.4)

[error] The HTML element with the interactive role 'link' is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like 'link'.
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)